crates/generator/src/generator.rs

```diff
@@ -775,6 +775,10 @@ impl Generator {
 
         Ok((output, data))
     }
+
+    pub fn get_backend_info(&self) -> Vec<gw_common::backend::BackendInfo> {
```

Comment:
The get_backend function is enough, we can call get_backend in the RPC module and convert it to BackendInfo.